build/__sdkStorageConstants.sh

```diff
@@ -3,6 +3,10 @@
 ENABLE_DYNAMIC_INSTALL_KEY='ENABLE_DYNAMIC_INSTALL'
 SDK_STORAGE_BASE_URL_KEY_NAME='ORYX_SDK_STORAGE_BASE_URL'
 TESTING_SDK_STORAGE_URL_KEY_NAME='ORYX_TEST_SDK_STORAGE_URL'
+PRIVATE_STAGING_STORAGE_SAS_TOKEN_KEY='ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN'
+ORYX_KEYVAULT_URI='https://oryx.vault.azure.net'
+STAGING_STORAGE_SAS_TOKEN_KEYVAULT_SECRET_NAME='ORYX-SDK-STAGING-PRIVATE-SAS-TOKEN'
+PRIVATE_STAGING_SDK_STORAGE_BASE_URL='https://oryxsdksstaging.blob.core.windows.net'
 DEV_SDK_STORAGE_BASE_URL='https://oryxsdksdev.blob.core.windows.net'
```

Comment:
Do we need to maintain this `DEV_SDK_STORAGE_BASE_URL` anymore in the repo if we're treating the staging storage account as the new dev?